sql/moz-fx-data-shared-prod/accounts_backend_external/dataset_metadata.yaml

```diff
@@ -0,0 +1,11 @@
+friendly_name: Firefox Accounts Backend External
+description: |-
+  Data extracted from the FxA backend services databases.
+  See https://mozilla.github.io/ecosystem-platform/reference/database-structure#database-fxa for more information.
+dataset_base_acl: restricted
+user_facing: false
+labels: {}
+workgroup_access:
```

Comment:
This dataset configuration is [nonstandard](Data+Access+Workgroups#DataAccessWorkgroups-Exampledatasetwithuniformspecificworkgroupaccess(creatingadatasetthatisaccessibletoaspecificworkgroup)) in that it's granting broad DE access to a dataset that is `restricted` (workgroup-confidential). There's some discussion of this in https://mozilla-hub.atlassian.net/browse/DSRE-1023, and an example of how we've approached that elsewhere in https://mozilla-hub.atlassian.net/browse/DSRE-1023?focusedCommentId=771018.

At any rate, because this configuration is nonstandard it requires matching configuration in https://github.com/mozilla-services/cloudops-infra/blob/master/projects/data-shared/tf/prod/envs/prod/bigquery-new/namespaces.auto.tfvars.json. If you're sure it's acceptable for DE to have this access (looks like that's how we have some other `_external` datasets set up) I can update ops logic to include an entry to allow this to deploy.